contracts/vendor/starkgate-contracts/.github/workflows/npm-publish.yaml

```diff
@@ -0,0 +1,79 @@
+name: Straknet bridge abi CI/CD
```

Comment:
Hmm looks like the submodule wasn't added correctly so all the code got pulled in?